### PR TITLE
Uses the threadsafe branch with threadlocals

### PR DESCRIPTION
**Pull request by csaunders**
This is just to verify that the gem is still working, once
https://github.com/Shopify/activeresource/pull/2 is merged
this will either point to that resulting ref or to the gem
when it gets released on rubygems.

Please review @boourns @benjlcox 

/cc @chrisbutcher @peterjm

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,4 +2,4 @@ source "https://rubygems.org"
 
 gemspec
 
-gem "activeresource", "~> 4.0.0"
\ No newline at end of file
+gem "activeresource-threadsafe", github: 'Shopify/activeresource'
```
Comment (peterjm) [inline Gemfile]: Making `shopify_api` *require* our fork of activeresource is a pretty major change.

If we decide to go this way, this probably requires a major version bump of `shopify_api`

**Comment by csaunders** [comment]
We got green!!!!!

**Comment by peterjm** [comment]
Looks like this is ready for review @boourns @benjlcox

**Comment by benjlcox** [comment]
Sorry for the delay! lgtm

**Comment by boourns** [comment]
LGTM :sheep:

**Pull request merged**